src/main/java/bisq/asset/tokens/Tari.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * CryptoTari is a stellar token issued by account:
+ * GD7UVDDJHJYKUXB4SJFIC6VJDQ4YADQCMRN3KLHJFV4H6NIUAEREVCO7
+ * block explorer: https://stellarchain.io/
+ */
+
+package bisq.asset.tokens;
+
+import bisq.asset.Token;
+import bisq.asset.StellarAddressValidator;
+
+public class Tari extends Token {
+
+    public Tari() {
+        super("CryptoTari", "TARI", new StellarAddressValidator() );
```

Comment:
Use existing regex validator.

Reply:
I have reworked to use the Regex validator, indeed it is indeed a better choice.